### DISCUSSION TITLE
PWR BI Sample Project 공유

### DISCUSSION THREAD

**Discussion by briankim1027** [fusiondevkr/pl900#41] [1 Q&A]
안녕하세요, BrianKim입니다.
작년에 서울디지털 재단에서 주관했었던 Power BI 과정 수강하면서 실습 프로젝트로 했었던 파일 입니다.
지난 차수에 Power BI를 다루었었는데요.
저도 작년에 재단 무료수업 들으면서 많이 배웠습니다.
PWR BI 서비스에서 공유하면 되는데 테넌트 설정이 개발자 계정이 아니고 그래서 복잡하더라구요.
파일이 커서 Zip으로 올렸는데.
주제는 
 - 2015년 이후 부터 가족의 세대 구성의 변화양상과 
 - 1/2인 세대가 많이 선호 이주하는 지역의 범죄 현황이 어떤지에 대한 분석 이었는데요.

범죄 현황에 대한 데이터 분석과 그래프/상관 등 분석은 PWR BI에서 하지 않고 파이선으로 하고 그래프를 캡쳐해서 붙여 넣은 겁니다.
참고하시기 바랍니다.
감사합니다.

[201208_[과제 제출]_1인 세대 & 범죄 현황 탐구_김혁.zip](https://github.com/fusiondevkr/pl900/files/7013196/201208_._1._.zip)

**Comment by justinyoo** [comment]
@briankim1027 아앗 감사합니다! ㅠㅠㅠ 👍👍👍👍👍

**Reply by sesme100** [reply]
다운로드 너무 잘 됩니다! 그리고 열자마자 데이터 정리가 너무 잘되어있고 시각화도 한눈에 정리가 잘 되어있어서 깜놀했습니다. 좋은 자료 공유해 주셔서 너무나 감사합니다!!

**Comment by briankim1027** [comment]
과제에 대한 해석 요약 정리 자료 입니다.
[1208_과제 발표 자료 요약_김혁.pptx](https://github.com/fusiondevkr/pl900/files/7070773/1208_._.pptx)

**Comment by briankim1027** [comment]
제가 분석한거 보다 더 깊게 서울 행정동별로 범죄현황을 분석한 결과가 최근에 대한범죄학회 연구지에 게재되었는데
그거를 다룬 기삽니다.
https://www.google.com/amp/s/m.khan.co.kr/local/Seoul/article/202108250601001/amp

**Reply by sesme100** [reply]
우와 감사합니다!!
근데 전해주신 URL로 들어가니 JSON파일로 나와요
